### PR TITLE
docs(slim): fixes typos and capitalizes Tensorflow

### PR DESCRIPTION
**Pull request by goldcaddy77**
Updates to docs:
- A few misspellings
- Grammar
- "tensorflow" -> "Tensorflow"

### CODE REVIEW

File: tensorflow/contrib/slim/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 TF-Slim is a lightweight library for defining, training and evaluating complex
 models in TensorFlow. Components of tf-slim can be freely mixed with native
-tensorflow, as well as other frameworks, such as tf.contrib.learn.
+Tensorflow, as well as other frameworks, such as tf.contrib.learn.
```
Comment (benoitsteiner) [inline tensorflow/contrib/slim/README.md]: Thanks for fixing these typo. Can you also capitalize the F in TensorFlow?

Comment (goldcaddy77) [inline tensorflow/contrib/slim/README.md]: (facepalm) On it.

**Comment by goldcaddy77** [comment]
@benoitsteiner - updated `(T|t)ensorflow` -> `TensorFlow` in this file as well as other *.md files.  Stayed out of .py, .java and .h comments though.  Lemme know if you'd rather I only update slim readme.

File: tensorflow/g3doc/api_docs/python/client.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ the session constructor.
 
 
 *  <b>`target`</b>: (Optional.) The execution engine to connect to.
-    Defaults to using an in-process engine. See [Distributed Tensorflow]
```
Comment (vrv) [inline tensorflow/g3doc/api_docs/python/client.md]: This is autogenerated from the python documentation, so you'll want to edit the version there instead and revert this file

File: tensorflow/g3doc/api_docs/python/functions_and_classes/shard0/tf.py_func.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ### `tf.py_func(func, inp, Tout, stateful=True, name=None)` {#py_func}
 
-Wraps a python function and uses it as a tensorflow op.
```
Comment (vrv) [inline tensorflow/g3doc/api_docs/python/functions_and_classes/shard0/tf.py_func.md]: same with this file

File: tensorflow/g3doc/api_docs/python/functions_and_classes/shard6/tf.train.Supervisor.md [hunk 4]
```diff
@@ -92,7 +92,7 @@ following values for the --master flag:
 
 * Specifying `'grpc://hostname:port'` requests a session that uses
   the RPC interface to a specific , and also allows the in-process
-  master to access remote tensorflow workers. Often, it is
+  master to access remote TensorFlow workers. Often, it is
```
Comment (vrv) [inline tensorflow/g3doc/api_docs/python/functions_and_classes/shard6/tf.train.Supervisor.md]: and this one

File: tensorflow/g3doc/api_docs/python/functions_and_classes/shard8/tf.Session.md [hunk 5]
```diff
@@ -68,7 +68,7 @@ the session constructor.
 
 
 *  <b>`target`</b>: (Optional.) The execution engine to connect to.
-    Defaults to using an in-process engine. See [Distributed Tensorflow]
+    Defaults to using an in-process engine. See [Distributed TensorFlow]
```
Comment (vrv) [inline tensorflow/g3doc/api_docs/python/functions_and_classes/shard8/tf.Session.md]: and this

File: tensorflow/g3doc/api_docs/python/script_ops.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ TensorFlow operators.
 
 ### `tf.py_func(func, inp, Tout, stateful=True, name=None)` {#py_func}
 
-Wraps a python function and uses it as a tensorflow op.
+Wraps a python function and uses it as a TensorFlow op.
```
Comment (vrv) [inline tensorflow/g3doc/api_docs/python/script_ops.md]: and this one, and python/train.md.

All the other are changes seem valid to me, so thanks!

**Comment by vrv** [comment]
Closing due to lack of response, but feel free to make the fixes and ping back and we'll reopen.  Trying to keep the number of active PRs down so things don't fall through the cracks

**Pull request closed without merging**